publishing-artifacts.md

```diff
@@ -7,33 +7,36 @@ You may obtain a copy of the License at
+ However, the development branch (`develop`) of [pravega-samples](https://github.com/pravega/pravega-samples/tree/develop) are likely to be unstable due to the Pravega/Flink connector snapshot artifact dependency. (_As the development branch of Pravega/Flink connector could possibly introduce a breaking change_).
 
-To overcome this issue, the Pravega/Flink connector snapshot artifacts are published to `jcenter` repository with a **stable** version label which can be referred by any downstream projects. This will guarantee to avoid any breaking changes from Pravega/Flink connector but at the same time, it is the responsibility of the downstream projects to sync with the latest shapshot updates.
+A typical version label of a snapshot artifact will have the label `-SNAPSHOT` associated with it (for e.g., `0.1.0-SNAPSHOT`). There could be more revisions for a snapshot version and the `maven/gradle` build scripts is responsible for fetching the most recent version from the available list of revisions. The snapshot repositories are usually configured to discard the old revisions based on some retention policy settings. Any downstream projects that are depending on snapshots are faced with the challenges of keeping up with the snapshot changes since they could possibly introduce any breaking changes.
 
+To overcome this issue, the Pravega/Flink connector snapshot artifacts are published to `jcenter` repository with a **stable** version label which can be referred by any downstream projects. This will guarantee to avoid any breaking changes from Pravega/Flink connector, but at the same time, it is the responsibility of the downstream projects to sync with the latest snapshot updates.
 
-# Publishing Snapshots
 
-The snapshots are the artifacts that are coming from `master` branch (development branch). The snapshot artifacts are published automatically to `jcenter` through Travis build setup. Any updates to `master` branch will trigger a build that will publish the artifacts upon succesful completion of the build. 
+## Publishing Snapshots
 
-We use `bintray` account to manage the artifact publishing. Here is the gradle task that is used to publish the artifacts. The published artifacts can be found here "https://oss.jfrog.org/jfrog-dependencies/io/pravega/pravega-connectors-flink_2.11/"
+The snapshots are the artifacts that are coming from `master` branch (development branch). The snapshot artifacts are published automatically to `jcenter` through Travis build setup. Any updates to `master` branch will trigger a build that will publish the artifacts upon successful completion of the build.
+
+We use `bintray` account to manage the artifact publishing. Here is the gradle task that is used to publish the artifacts. Please refer to the [published artifacts](https://oss.jfrog.org/jfrog-dependencies/io/pravega/pravega-connectors-flink_2.11/) for more information.

```

Comment:
Let's change it like below.

> 
We use `bintray` account to publish snapshot artifacts. Here is the [link](https://oss.jfrog.org/jfrog-dependencies/io/pravega/pravega-connectors-flink_2.11/) to the published snapshot artifacts. The gradle task that is used to publish the artifacts is provided below.